l10n/en/footer-refresh.ftl

```diff
@@ -4,12 +4,13 @@
 
 footer-refresh-discover-mozilla-products =Discover { -brand-name-mozilla } products and initiatives. We promise to keep your email private and secure — no sharing, no selling, just great updates.
 footer-refresh-leadership = Leadership
-footer-refresh-advertise = Advertise with { -brand-name-mozilla }
-footer-refresh-mozilla-advertising = { -brand-name-mozilla } Advertising
+footer-refresh-mozilla-ads = { -brand-name-mozilla-ads }
 # The content inside the <span> is added to provide information for screen reader users only, and it is visually hidden
-footer-refresh-learn-more-about-mozilla-advertising = Learn more <span>about { footer-refresh-mozilla-advertising }</span>
+footer-refresh-learn-more-about-mozilla-ads = Learn more <span>about { footer-refresh-mozilla-ads }</span>
+# Obsolete string (expires 2026-10-03)
+footer-refresh-learn-more-about-mozilla-advertising = Learn more <span>about { -brand-name-mozilla } Advertising</span>
 footer-refresh-firefox-release-notes = { -brand-name-firefox } Release Notes
-footer-refresh-privacy-first = Privacy-first advertising solutions for brands, publishers, and platforms.
+footer-refresh-privacy-first = Extend reach and deepen trust.
```

Comment:
IIUC this update won't propagate to other languages without changing the key as it's been around for a while already: https://github.com/mozilla-l10n/www-l10n/blob/23bf497/en-CA/footer-refresh.ftl#L10

I see the sync PR approved in l10n so this may very well be deliberate, just leaving a note in case it's not.

Reply:
Maureen and I agreed to do the fast-follow before shipping https://github.com/mozilla/bedrock/pull/16780/files